src/server/pfs/server/driver.go

```diff
@@ -298,11 +300,8 @@ func (d *driver) listRepo(pachClient *client.APIClient, includeAuth bool) (*pfs.
 }
 
 func (d *driver) deleteRepo(txnCtx *txnenv.TransactionContext, repo *pfs.Repo, force bool) error {
-	if err := d.storage.Store().Walk(txnCtx.ClientContext, repo.Name, func(p string) error {
-		return d.storage.Delete(txnCtx.ClientContext, p)
-	}); err != nil {
-		return err
-	}
+	// TODO: impoement deleteRepo with commitStore
```

Comment:
This misspell got a chuckle out of me. Idempotent?

Reply:
implement

Reply:
This is now implemented with a note to switch to transactions.